scripts/load_osf.py

```diff
@@ -0,0 +1,25 @@
+import requests
+from pathlib import Path
+
```

Comment:
Can you add annotation?

Reply:
What do you mean with annotation? Documentation?